lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -228,32 +235,44 @@ def self.upload_binary(app_id, binary_path)
         connection.post("/app-binary-uploads?app_id=#{app_id}", File.open(binary_path).read) do |request|
           request.headers["Authorization"] = "Bearer " + auth_token
         end
+      rescue Faraday::ResourceNotFound
+        UI.crash!(ErrorMessage::INVALID_APP_ID)
+      rescue Errno::ENOENT
+        UI.crash!(ErrorMessage::APK_NOT_FOUND)
       end
 
       def self.upload(app_id, binary_path)
         upload_token = get_upload_token(app_id, binary_path)
-        status = upload_status(upload_token, app_id)
+        status = upload_status(app_id, upload_token)
```

Comment:
Howcome you decided to reorder the params?  No biggie just curious what your thoughts were.

Reply:
Just to make it consistent with all the other method headers which have app_id as the first arg